naturalspeech2_pytorch/naturalspeech2_pytorch.py

```diff
@@ -109,7 +115,16 @@ def compute_pitch(spec, sample_rate, hop_length, pitch_fmax=640.0):
     )
     f0 = pw.stonemask(spec.astype(np.double), f0, t, sample_rate)
     return f0
-
+def f0_to_coarse(f0):
```

Comment:
is there any reason `f0` would ever be in numpy?

i think we can just use torch and simplify conditionals below

although i do not understand the mathematics here, the `1127`, `700` can also be named and turned into keyword arguments for the `f0_to_coarse`

Reply:
if we use this with compute_pitch it'll be in numpy because pw give numpy array as output.

Reply:
ohh yes, you are correct, that sounds good!